### DISCUSSION TITLE
Internal InnerSource Ambassadors/Champions Program

### DISCUSSION THREAD

**Discussion by dellagustin-sap** [InnerSourceCommons/InnerSourcePatterns#729] [Pattern Ideas]
To be refined, we are planning to do it at SAP, we already have an Open Source Champions program that we want to extent it or mirror it to InnerSource.

This is also something that was mentioned on the summit https://innersourcecommons.org/events/isc-2024/ at the talk "The InnerSource Flywheel".

@rrrutledge could share some experience here as well.

**Comment by dellagustin** [comment]
Related issue on the ISPO WG board: https://github.com/InnerSourceCommons/ispo-working-group/issues/137

Public Reference from SAP: https://community.sap.com/t5/open-source-blogs/sap-open-source-champions/ba-p/13539587

**Comment by dellagustin-sap** [comment]
A first version of this pattern was integrated as Initial by @spier : https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/5004c9cc05e443ddf077247a1a596bb5b6aded64/patterns/1-initial/innersource-ambassador.md

**Reply by dellagustin-sap** [reply]
My ongoing review notes of the draft:
> Patlet
> When driving InnerSource adoption through a large, decentralized organization it is hard to understand and address the local challenges that come up in different departments and regions. Local volunteers, called InnerSource Ambassadors, provide localized support by promoting InnerSource principles and acting as a communication bridge between their teams and the ISPO.

An additional challenge to be mentioned here is that it is hard for a central team driving InnerSource to reach all parts of the organization for distributing knowledge and raising awareness.

**Reply by dellagustin-sap** [reply]
My ongoing review notes of the draft:
> Story
> In a multinational organization, the ISPO implemented InnerSource initiatives but noticed uneven adoption across departments. After appointing a local ambassador in one department, adoption rates improved as the ambassador tailored practices to the department's context and provided valuable feedback to the ISPO.

There are multiple dimensions where we could have ambassador, this story explore two: locations (_In a multinational organization_) and departments (organizational hierarchy).

Should we have both represented in the story, and if yes, should we adapt it to better represent it?

e.g.:
> In a **multinational and/or hierarchical organization**, the ISPO implemented InnerSource initiatives but noticed uneven adoption across **locations and/or departments**. After appointing a local ambassador in one department, adoption rates improved as the ambassador tailored practices to the department's context and provided valuable feedback to the ISPO.

or just:
> In a **(deeply) hierarchical organization**, the ISPO implemented InnerSource initiatives but noticed uneven adoption across departments. After appointing a local ambassador in one department, adoption rates improved as the ambassador tailored practices to the department's context and provided valuable feedback to the ISPO.

**Reply by dellagustin-sap** [reply]
PR for adding incentives to the solutions: https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/759

**Reply by dellagustin-sap** [reply]
List for PRs created during our working sessions today:

- https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/759
- https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/758
- https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/757
- https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/756

**Reply by spier** [reply]
Amazing! Looks like this was a really productive session. Already merged 3/4 PRs.

Aim to look at the last PR over the weekend.

**Reply by spier** [reply]
The PR #758 is a bit harder to merge as is, however it started some good exploration.

Left some feedback here:
https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/758#pullrequestreview-2542198230

**Comment by rrrutledge** [comment]
What do the terms "Ambassador" and "Champion" mean?  Are they synonyms?  Or a distinct meaning?

**Reply by sallydeering1** [reply]
Ambassadors are 'support people' and 'advocates' (using Jono Bacon's typical community personas) where support people help other [innersource practitioners] solve problems.  For example, they may do personal mentoring or provide innersource training for groups of people.  And advocates want the [innersource] community to succeed. They do marketing, outreach and promotion of [innersourcing] to raise awareness and to get people interested and involved in it.  I'm not proposing introducing new terms of support or advocate but it helps articulate what ambassadors do.  I'm still not clear on what IS Champions would do vs. IS Ambassadors but open to learning.

**Reply by spier** [reply]
When writing the pattern I used champion as a synonym for ambassador.
I believe that both terms where used in the description when the pattern was initially proposed.
